package.json

```diff
@@ -45,7 +46,9 @@
     "test:visual:native": "npm run build && playwright test --grep @visual-regression",
     "test:visual:refresh": "scripts/run-playwright-container.sh bash -c 'npm install && npm run test:visual:native -- --update-snapshots'",
     "test:watch": "vitest",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "release:alpha": "release-it --preRelease=alpha",
+    "release:stable": "release-it --github.draft"
```

Comment:
Unfortunately the `--git.tagMatch` value that was here before (from an example in the Release It! docs) didn't actually work as it was a glob and `*` matches anything. As far as I can see, it's not possible to use a standard glob to match non-pre-release versions in a generic fashion. 

Therefore, I've set it to leave the release as a draft for stable releases for now (we'll need something different for the first stable release anyway).